setup.py

```diff
@@ -15,7 +15,15 @@
 with io.open("README.md", "rt", encoding="utf8") as f:
     readme = f.read()
 
-requires = ["rlp", "coincurve", "dataclasses; python_version < '3.7'", "pycrypto", "jsondatabase"]
+requires = [
+    "rlp",
+    "coincurve",
+    "dataclasses; python_version < '3.7'",
+    "pycrypto",
+    "jsondatabase",
+    "json-rpc",
```

Comment:
It is a library to make a server. I think you've followed their client library, but it doesn't contain any dependencies from their package. It only requests to the server through `requests`, the HTTP request library, and dump/load with built-in `json` library'
https://github.com/pavlov99/json-rpc/blob/master/examples/client.py

I've tried to use `jsonrpcclient[requests]` before: https://jsonrpcclient.readthedocs.io/en/latest/api.html